src/devices/IS31FL3730/README.md

```diff
@@ -0,0 +1,37 @@
+﻿# IS31FL3730 - I2C LED Matrix Controller
+The IS31FL3730 is a flexible I2C-based LED Matrix controller.  Each IS31FL3730 can control a pair of LED matrices in
+a variety of layouts.
+
+## Datasheet
+[IS31FL3730](https://cdn.hackaday.io/files/1692447240935296/IS31FL3730.pdf)
+
+## Binding Notes
+Create a new IS31FL3730 driver instance:
+
+```cs
```

Comment:
better use `csharp` as it's the official way of writing a C# code block in Markdown